nrf_rpc/CHANGELOG.rst

```diff
@@ -19,6 +19,11 @@ Added
 
 * Added the :kconfig:option:`CONFIG_NRF_RPC_ZCBOR_BACKUPS` Kconfig option for setting up the number of zcbor backups.
 
+Changes
+=======
+
+* Different transport mechanisms are initialized independent of each other.
```

Comment:
Could you make this entry a bit more precise: e.g.:
```suggestion
* Added a possibility for transport in each group to initialize independently of other groups. This enables the RPC to work for groups which are ready earlier rather than waiting for all groups to initialize their transports.
```
or something similar.

Reply:
Thanks for the suggestion.
I have changed the second part of the entry. Because the `nrf_rpc_init` function waits for all groups to be initialized. After that it is possible to use RPC.